app/views/unattended/provisioning_templates/snippet/kickstart_network_interface.erb

```diff
@@ -61,11 +61,7 @@ if subnet6
     network_options.push("--ipv6=#{@iface.ip6}/#{subnet6.cidr}")
     network_options.push("--ipv6gateway=#{subnet6.gateway}")
   elsif subnet6.dhcp_boot_mode?
-    if @use_slaac
```

Comment:
The `use_slaac` option is now unused so it should be removed, as well as the `use-slaac` parameter. The alternative is to default the parameter to `true`.

Reply:
I think removing the parameter will be less confusing than having 
``` ruby
use_slaac = hoast_parameter_true?('use-slaac', true)
```
WDYT?